### PR TITLE
added/fixed field length counters

### PR DESCRIPTION
**Pull request by djcooke**
Field length counters were half-implemented on most create/edit pages, working on a few, and not present at all on a few. Added/fixed them on all pages

### CODE REVIEW

File: miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditGroupController.java [hunk 1]
```diff
@@ -80,6 +90,11 @@ public void setSecurityManager(SecurityManager securityManager) {
       throw ex;
     }
   }
+  
+  @ModelAttribute("maxLengths")
+  public Map<String, Integer> maxLengths() throws IOException {
+    return DbUtils.getColumnSizes(interfaceTemplate, "_Group");
```
Comment (frankie13) [inline miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditGroupController.java]: These are kinda gross and should probably be defined in an enum or similar:

```java
public enum Tables {
    Box("Box"),
    Group("_Group"),
    Pool("Pool"),
    KitDescriptor("KitDescriptor");

    private final String name;       

    private Tables(String s) {
        name = s;
    }

    public String toString() {
       return this.name;
    }
}
```

**Comment by frankie13** [comment]
:+1:  this all seems good to me.  Would be nice to get the table names somewhere standardised though.

Thanks!

**Comment by apmasell** [comment]
:+1:

**Comment by djcooke** [comment]
@frankie13 I had followed the existing method of doing this, but I agree completely. I will create a separate ticket to remove hard-coded table names (and probably JDBC calls) from the Controller classes

**Pull request merged**